package.json

```diff
@@ -60,6 +61,7 @@
     "react-visjs-timeline": "^1.6.0",
     "remark-gfm": "^4.0.1",
     "server-only": "^0.0.1",
+    "styletron-engine-atomic": "^1.6.2",
```

Comment:
Not completely sure how this library got added here, we don't depend directly on the Styletron engine in our code. We instead use Styletron through Baseweb's wrapper.